lucien-ios/ProfileViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ProfileViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 9/27/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController {
+
+    @IBOutlet weak var emptyProfilePicture: UIImageView!
+    @IBOutlet weak var logoutButton: UIButton!
+    @IBOutlet weak var nameTitle: UILabel!
+    @IBOutlet weak var name: UILabel!
+    @IBOutlet weak var emailTitle: UILabel!
+    @IBOutlet weak var email: UILabel!
+    @IBOutlet weak var profileTitle: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        emptyProfilePicture.contentMode = .scaleAspectFit
+        let dark = UIColor(redVal: 43, greenVal: 45, blueVal: 66, alphaVal: 1.0)
```

Comment:
Instead of defining this here, add it to `LucienTheme` in the same way that the other colors in there are.